test/marked-element.html

```diff
@@ -241,6 +241,21 @@
 
     });
 
+    suite('events', function() {
+      var markedElement;
+
+      setup(function() {
+        markedElement = fixture('CamelCaseHTML');
+      });
+
+      test('render() fires marked-render-complete', function(done) {
+        markedElement.addEventListener('marked-render-complete', function() {
+          done();
```

Comment:
I was following the pattern at https://github.com/PolymerElements/seed-element/blob/master/test/basic-test.html#L54

Given that there's no `event.detail` associated with the event, I'm not sure what I'd assert, so yeah, it will rely on a timeout to trigger the failure case.